coordinator/src/main/scala/com/socrata/datacoordinator/common/util/DatasetIdNormalizer.scala

```diff
@@ -0,0 +1,26 @@
+package com.socrata.datacoordinator.common.util
+
+import com.ibm.icu.text.Normalizer
+import com.rojoma.json.v3.io.{IdentifierEvent, FieldEvent, StringEvent, JsonEvent}
+
+object DatasetIdNormalizer {
+
+  val normalizationMode: Normalizer.Mode = Normalizer.NFC
```

Comment:
We should never need to change it; it doesn't matter what normalizer we use, just that it's _some_ single one. The point of it is just to ensure that "á" and "á" always compare equal to the less intelligent sorts of string comparators whether they come in as LATIN_SMALL_LETTER_A_WITH_ACUTE or LATIN_SMALL_LETTER_A and COMBINING_ACUTE_ACCENT.